versions/3.0.md

```diff
@@ -391,14 +391,14 @@ All objects defined within the components object will have no effect on the API
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="definitionsObject"></a> | [Definitions Object](#definitionsObject) | A hash containing payload definitions for the specification.
-<a name="responsesDefinitionsObject"></a> | [Responses Definitions Object](#responsesDefinitionsObject) | Reusable responses objects.
-<a name="parametersDefinitionsObject"></a> | [Parameters Definitions Object](#parametersDefinitionsObject) | An object to hold parameters to be reused across operations. Parameter definitions can be referenced to the ones defined here.
-<a name="requestBodyDefinitionsObject"></a> | [Request Body Definitions Object](#requestBodyDefinitionsObject) | An object to hold request body definitions to be reused across operations. Request Body definitions can be referenced to the ones defined here.
-<a name="responseHeadersDefinitionsObject"></a> | [Response Headers Definitions Object](#responseHeadersDefinitionsObject) | Response headers to reuse across the specification.
-<a name="securityDefinitionsObject"></a> | [Security Definitions Object](#securityDefinitionsObject) | Security definitions to reuse across the specification.
-<a name="linksDefinitionObject"></a> | [Link Definitions Object](#linksDefinitionsObject) | Link definitions to reuse across the specification.
-<a name="callbacksDefinitionObject"></a> | [Callback Definitions Object](#callbacksDefinitionsObject) | Callback definitions to reuse across the specification.
+<a name="componentsObject-definitionsObject"></a> | [Definitions Object](#definitionsObject) | A hash containing payload definitions for the specification.
```

Comment:
```
<a name="componentsDefinitionsObject"></a>definitions | [Definitions Object](#definitionsObject) | A hash containing payload definitions for the specification.
```
in this case (notice the anchor and the added field name).

The first header of the table, `Field Pattern` should be `Field Name`.